README.md

```diff
@@ -114,13 +114,21 @@ At the end in the controller, you will have true json, xml for payload & integer
 
 ### Automatic binding route
 
-**RREST** bind the route to a controller by following this convention:
+**RREST** binds the route to a controller by following this convention:
 * `POST /item/` -> `Controllers\Item#postAction`
 * `GET /item/{itemId}/` -> `Controllers\Item#getAction`
 * `GET /item/{itemId}/comment` -> `Controllers\Item\Comment#getAction`
 * `PUT /item/{itemId}/comment/{commentId}` -> `Controllers\Item\Comment#putAction`
 
-> note you can define the controller namespace (here Controllers) in the RREST\RREST constructor
+> Note: You can define the controller namespace (here Controllers) in the RREST\RREST constructor
+
+> Note: Currently, the order of API definitions matters. See this as an example :
+> ```raml
+> /item/{itemId}
+> /item/archive
```

Comment:
I would add a working example, specify that this is a best practice and warn about the side effect if not applied.

Reply:
Thanks, I've rephrased it to emphasis good practice.